src/components/GraphFilter/GraphFilter.tsx

```diff
@@ -70,6 +57,7 @@ export default class GraphFilter extends React.Component<GraphFilterProps, Graph
   };
 
   render() {
+    console.log(this.props.graphDuration.value);
```

Comment:
Just wondering, is that an deliberate console.log or perhaps you forgot it?

Reply:
Changed !!! Thanks @xeviknal